### DISCUSSION TITLE
Can't get mouse or keyboard input working in samples

### DISCUSSION THREAD

**Discussion by dschmidt004** [Voltstro-Studios/UnityWebBrowser#140] [Q&A] [answered]
I've imported your samples to a blank unity environment but still can't get keyboard and mouse input to work while running... I'm able to see and change the initial URL and the websites load but I have no inputs...

**Comment by Voltstro** [comment, marked as answer]
Your probably missing Unity's event system. You can add the default one by going (Right Click on Hierarchy) -> UI -> Event System

**Reply by dschmidt004** [reply]
You're a genius, Thank you!
